tensorflow/python/ops/image_ops_impl.py

```diff
@@ -511,6 +512,77 @@ def _rot270():
   result.set_shape([shape[0], None, None, shape[3]])
   return result
 
+@tf_export('image.rot90ND')
+def rot90ND(image, rot_dims, k=1, name=None):
+  """Rotate image(s) counter-clockwise by 90 degrees.
+
+  Args:
+    image: N-D Tensor where N >= 3 
+    k: A scalar integer. The number of times the image is rotated by 90 degrees.
+    rot_dims: A list of integers with length 2. The dimensions to be rotated. 
```

Comment:
Instead of a list of length 2 (and checking that), I wonder if it would be better to just take two scalars. So something like:

```python
def rot90ND(image, dim0, dim1, k=1, name=None):
```

(Or perhaps `height_dimension` and `width_dimension` instead of `dim0` and `dim1`)